src/wrappers/themis/rust/src/lib.rs

```diff
@@ -17,7 +17,7 @@
 //! **Themis** is a high-level cryptographic library.
 //!
 //! Themis allows you to protect data at rest (in your database, files, or wherever you store it
-//! in your application) and data in motion (i.e. travelling between client and server, server
+//! in your application) and data in motion (i.e. traveling between client and server, server
```

Comment:
that's just british spelling here
https://www.merriam-webster.com/words-at-play/traveling-vs-travelling-usage

```suggestion
//! in your application) and data in motion (i.e. travelling between client and server, server
```